### PR TITLE
General improvements on sbom and provenance

### PR DESCRIPTION
**Pull request by pjbgf**
- Add `--platform` for both `sbom` and `provenance` to return platform specific data.
- Add resolved dependencies and invocationID information when converting provenance data from v0.2 to v1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ slsactl download provenance rancher/cis-operator:v1.0.15
 slsactl download provenance --format=slsav1 rancher/cis-operator:v1.0.15
 ```
 
+By default, the returned provenance would be for `linux/amd64`. To select a
```
Comment (macedogm) [inline README.md:26]:
```suggestion
By default, the returned provenance will be for `linux/amd64`. To select a
```

File: README.md [hunk 2]
```diff
@@ -37,6 +40,9 @@ If Cyclonedx is required instead:
 slsactl download sbom -format cyclonedxjson rancher/cis-operator:v1.0.15
 ```
 
+By default, the returned provenance would be for `linux/amd64`. To select a
```
Comment (macedogm) [inline README.md:43]:
```suggestion
By default, the returned provenance will be for `linux/amd64`. To select a
```

File: cmd/root.go [hunk 3]
```diff
@@ -19,6 +19,7 @@ var (
 
 Available commands:
   download:   Download artefacts from container image
+  verify: 	  Verifies the container image's signature
```
Comment (pdellamore) [inline cmd/root.go:22]: I guess there is a exceeding `tab` here.

Comment (macedogm) [inline cmd/root.go:22]: Perhaps there is an extra tab here?

**Review by pdellamore — APPROVED** [review]

**Review by macedogm — APPROVED** [review]

**Review by tashima42 — APPROVED** [review]

**Pull request merged**